### ISSUE TITLE
Replace bottom sheet with the android context menu

### ISSUE THREAD

**Issue by Z12aS** [Aliucord/plugin-requests#310]
### Plugin Description

title pretty much

### Request Agreement

- [X] I did indeed check to make sure my plugin request is original!

**Comment by Vendicated** [comment]
I don't think this is possible. Emoji Picker and User Profiles are two examples of bottomsheets, how in the world would you show those with an android context menu?

also whats the motive behind this? whats wrong with bottom sheets?

**Comment by C10udburst** [comment]
iirc it is possible to open `BottomSheet` in fullscreen with `openPageWithProxy`. it's not what the issue author had in mind but at least possible
